little correction

boufni95 opened the pull request:
the word "all" was written twice

boufni95 committed:
little correction

the word "all" was written twice

renepickhardt reviewed (approved):
Thanks! Don't forget to add yourself to the contributors list. If you don't do that to tomorrow I will merge without it. You could still do it later but it is easiest if it happens with the pull request

boufni95 committed:
edit contributors list

added my name to contributors list

boufni95 committed:
Merge pull request #1 from boufni95/develop

edit contributors list

renepickhardt committed:
put the contributor the the alphabetically correct place

mastering bitcoin has alphabetic ordering with the first letter of the first name or nick name
